apps/community/src/pages/co-authors.tsx

```diff
@@ -0,0 +1,446 @@
+    {
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+      refetchInterval: false,
+      cacheTime: 0,
+    }
+  )
+
+  useEffect(() => {
+    onUpdateTipsPanel(
+      <Trans
+        i18nKey="help_text"
+        t={t}
+        components={{
+          h3: <Heading as="h3" fontSize="16px" mt="32px" mb="12px" />,
+          p: <Text fontSize="14px" fontWeight="400" color="#737373;" />,
+        }}
+      />
+    )
+  }, [])
+
+  const isAdmin = useMemo(
+    () =>
+      data?.collaborators.some(
+        (item) => item.is_administrator && item.address === userProps?.address
+      ),
+    [userProps, data]
+  )
+  const bindList = data?.collaborators.filter((i) => !i.is_administrator) || []
+
+  const EmptyCaption =
+    bindList.length === 0 ? <TableCaption>{t('empty')}</TableCaption> : null
+
+  return (
+    <Container as={Grid} gridTemplateColumns="3fr 1fr" gap="20px">
+      <Flex
+        direction="column"
+        align="center"
+        {...cardStyleProps}
+        w="full"
+        h="full"
+        p="32px"
+      >
+        <Flex justifyContent="space-between" w="full">
+          <Heading fontSize="18px" lineHeight="20px">
+            {t('title')}
+          </Heading>
+        </Flex>
+        <Tabs w="full" variant="normal" mt="38px">
+          <TabList>
+            <Tab pl="0">{t('tabs.management')}</Tab>
+          </TabList>
+
+          <TabPanels>
+            <TabPanel p="32px 0">
+              <Text fontWeight={500}>{t('management_text')}</Text>
+              {isAdmin ? (
+                <BindButton
+                  refetch={() => refetch()}
+                  disabled={bindList.length >= 3}
+                />
+              ) : null}
+              <Box mt="32px">
+                {bindList.length >= 3 ? (
+                  <Text
+                    mt="20px"
+                    color="red"
+                    fontWeight="400"
+                    fontSize="12px"
+                    lineHeight="16px"
+                  >
+                    {t('unbind_limit')}
+                  </Text>
+                ) : null}
+
+                <TableContainer
+                  mt="8px"
+                  borderRadius="8px 8px 0px 0px"
+                  overflow="hidden"
+                  border="1px solid #F2F2F2"
+                >
+                  <Table variant="unstyled">
+                    {isLoading ? (
+                      <TableCaption>
+                        <Spinner />
+                      </TableCaption>
+                    ) : (
+                      EmptyCaption
+                    )}
+                    <Thead>
+                      <Tr>
+                        <Th>{t('wallet_address')}</Th>
+                        <Th>{t('state')}</Th>
+                        <Th>{t('operate')}</Th>
+                      </Tr>
+                    </Thead>
+
+                    <Tbody>
+                      {data?.collaborators.map((item) => {
```

Comment:
这里可以使用 filter, 不需要 return null
```suggestion
                      {data?.collaborators.filter(({ is_administrator: isAdminRemote }) => !isAdmin || !isAdminRemote).map((item) => {
```